src/libproc_macro/lib.rs

```diff
@@ -1199,7 +1300,7 @@ impl TokenTree {
             '#' => Pound,
             '$' => Dollar,
             '?' => Question,
-            _ => panic!("unsupported character {}", op),
+            _ => panic!("unsupported character {}", ch),
```

Comment:
Can this become `unreachable!()` now? As only a finite set of characters can be used.